### DISCUSSION TITLE
ダークモード時に記事を削除するダイアログの色がおかしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#437] [要望・アイデア・不具合]
白背景に白文字になってしまっています。

<img width="611" alt="スクリーンショット 2023-02-20 16 28 34" src="https://user-images.githubusercontent.com/20086673/220040004-f374ee69-f003-450f-9caf-c8a3ab169532.png">

ダイアログ内のテキストにフォーカスを当てると：

<img width="611" alt="スクリーンショット 2023-02-20 16 29 14" src="https://user-images.githubusercontent.com/20086673/220040069-752ea753-d36c-4359-a86c-e2b4d6cb3898.png">

**Comment by tanaka-asami415** [comment]
@munierujp 
ご報告ありがとうございます！ご不便をおかけして、申し訳ありません。
こちら対応中ですので、反映までお待ちいただけると幸いです🙇

**Comment by kimkim0814** [comment]
@munierujp
お待たせ致しました。
ダークモード時に記事を削除するダイアログの文字が見えない件について、修正が完了しました。
ご不便おかけいたしました。:bow:

**Reply by munierujp** [reply]
修正されていることを確認しました。ありがとうございました。
